src/components/cover/cover.tsx

```diff
@@ -28,7 +28,9 @@ export const Cover = ({
   shadow
 }: CoverProps) => {
   const [imageLoaded, setImageLoaded] = useState<boolean | null>(null);
-  const handleSetImageLoaded = () => setImageLoaded(true);
+  const handleSetImageLoaded = useCallback(() => {
+    setImageLoaded(true);
+  }, []);
```

Comment:
I appreciate the intent here but I wonder if this callback should in fact not have any dependencies at all - such as the `id` or `idType` which will effect the source of the final image. If you think it is OK then move along :smile:

Reply:
I am not sure if they should be there? 😅 If I put them in the dependency array I got this warning:
`React Hook useCallback has unnecessary dependencies: 'id' and 'idType'. Either exclude them or remove the dependency array.eslint`